docs/root/intro/arch_overview/upstream/aggregate_cluster.rst

```diff
@@ -0,0 +1,61 @@
+.. _arch_overview_aggregate_cluster:
+
+Aggregate Cluster
+=================
+
+Aggregate cluster is used for failover between clusters from different service discovery mechanism, e.g., 
+failover from EDS upstream cluster to STRICT_DNS upstream cluster. Aggregate cluster loosely couples multiple 
+clusters by referencing their name in the :ref:`configuration <envoy_api_msg_config.cluster.aggregate.ClusterConfig>`. 
+The fallback priority is defined implicitly by the ordering in the :ref:`clusters <envoy_api_field_config.cluster.aggregate.ClusterConfig.clusters>`.
+Aggregate cluster uses tiered load balancing. The load balancer chooses cluster and piority first and then deletgates the load balancing from that priority
```

Comment:
Is it really tiered, or is it linearize at load time and then regular priority/locality load balancing?

Reply:
It is really tiered. After linearing, only host and priority are selected. The load balancer will delegate the load balancing to the selected the cluster. The load balancer of aggregate cluster should respect the load balancing policy of including clusters.

Reply:
The doc shows users that the aggregate cluster does a two tier process instead of some internal implementation.